src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.component.tsx

```diff
@@ -73,28 +107,39 @@ export function BaseCellComponent({
     }
     return "thisMonth";
   }
+
   //  #region view
   return (
-    <td
-      className="mainCell"
-      id={getId()}
-      onClick={(): void => {
-        !isBlocked && onClick && onClick();
-      }}
-      onContextMenu={handleContextMenu}
-      onBlur={(e): void => {
-        onBlur && onBlur();
-      }}
-    >
-      {isPointerOn && !isBlocked && (
-        <InputComponent
-          className="cell-input"
-          onValueChange={(value): void => _onValueChange(value)}
-          onKeyDown={(e: React.KeyboardEvent<HTMLInputElement>): void => _onKeyDown(e)}
-        />
-      )}
-      {(!isPointerOn || (isPointerOn && isBlocked)) && <span>{value}</span>}
-    </td>
+    <>
```

Comment:
Czy ten fragment jest konieczny, skoro zwracamy tylko 1 komponent? Usunąłem go i działa tak samo, chyba, że czegoś nie rozumiem :)

Reply:
Nie, dzięki, nie zauważyłem tego :D
W którymś momencie implementacji potrzebowałem zwracać dwa i zapomniałem to usunąć